pipeline-adapters/mlrun-pipelines-kfp-v2/setup.py

```diff
@@ -41,7 +41,7 @@
     python_requires=">=3.9, <3.12",
     install_requires=[
         "kfp_server_api~=2.0.0",
-        "kfp[kubernetes]~=2.5.0",
+        "kfp[kubernetes]~=2.6.0",
```

Comment:
if we do the bump, lets make it `2.10.1` as this is the currently latest kfp sdk for clients